components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import { action } from "@storybook/addon-actions";
+import { select } from "@storybook/addon-knobs";
+import * as React from "react";
+import TextArea, { InputSize } from "./TextArea";
+
+export default {
+  title: "TextArea",
+  component: TextArea
+};
+
+export const example = () => (
+  <TextArea
+    size={select("size", SIZES, undefined)}
+    onChange={action("inputed")}
+  />
```

Comment:
It would be better if `placeholder`, `rows` and `fullWidth` are customizable with knobs.